integration-tests/api/factories/simple-sales-channel-factory.ts

```diff
@@ -1,6 +1,6 @@
 import { Connection } from "typeorm"
 import faker from "faker"
-import { SalesChannel } from "@medusajs/medusa"
+import { SalesChannel } from "@medusajs/medusa/dist/models/sales-channel"
```

Comment:
question: Should be left unchanged no?

Reply:
Indeed, and if there is an error for the import it is most likely that we have to rebuild and re scan the deps